aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -898,6 +898,7 @@ def build(self, event: ResponseEventT, cors: CORSConfig | None = None) -> dict[s
             **event.header_serializer().serialize(headers=self.response.headers, cookies=self.response.cookies),
         }
 
+T_route = TypeVar("T_route", bound=Callable[..., Any])
```

Comment:
Please check if importing `AnyCallableT` from `aws_lambda_powertools/shared/types.py` works the same.

Reply:
Hi @leandrodamascena, I did the import and it seems to be working!